docs/source/notebooks/tpt.ipynb

```diff
@@ -0,0 +1,779 @@
+   "source": [
+    "fig, axes = plt.subplots(1, 2, figsize=(18, 10))\n",
+    "dividers = [make_axes_locatable(axes[i]) for i in range(len(axes))]\n",
+    "caxes = [divider.append_axes(\"right\", size=\"5%\", pad=0.05) for divider in dividers]\n",
+    "\n",
+    "titles = [\"Gross flux\", \"Net flux\"]\n",
+    "fluxes = [flux.gross_flux.m, flux.net_flux.m]\n",
+    "\n",
+    "cmap = plt.cm.copper_r\n",
+    "thresh = [0, 1e-12]\n",
+    "\n",
+    "for i in range(len(axes)):\n",
+    "    ax = axes[i]\n",
+    "    F = fluxes[i]\n",
+    "    ax.set_title(titles[i])\n",
+    "\n",
+    "    vmin = np.min(F[np.nonzero(F)])\n",
+    "    vmax = np.max(F)\n",
+    "\n",
+    "    sim.plot_2d_map(ax)\n",
+    "    sim.plot_network(ax, F, cmap=cmap, connection_threshold=thresh[i])\n",
+    "    norm = mpl.colors.LogNorm(vmin=vmin, vmax=vmax)\n",
+    "    fig.colorbar(plt.cm.ScalarMappable(norm=norm, cmap=cmap), cax=caxes[i]);"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "From the gross flux one can derive the _total flux_, the total number of reactive $A\\rightarrow B$ trajectories per time unit\n",
+    "\n",
+    "$$\n",
+    "f_{\\mathrm{tot}}^{AB} = \\sum_{i\\in A, j\\not\\in A}f_{ij}^{AB},\n",
+    "$$\n",
+    "\n",
+    "which can be accessed by:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f\"Total flux = {flux.total_flux.m:.3e}/step\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This quantity gives rise to the *total transition rate*, which is the number of events given we start in $A$\n",
+    "\n",
+    "$$\n",
+    "k_{AB} = \\frac{f_\\mathrm{tot}^{AB}}{\\sum_i\\pi_iq_i^{(-)}}\n",
+    "$$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f'Rate = {flux.rate.m:.3e}/step')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "which is also the inverse $A\\to B$ mean first passage time $\\mathrm{mfpt} = 1/k_{AB}$"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "print(f'MFPT = {flux.mfpt.m:.3f} steps')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Coarse-graining of fluxes\n",
+    "\n",
+    "For better interpretability, one can cluster microstates defined in a Markov state model into metastable sets using, e.g., [PCCA+](../notebooks/pcca.ipynb).\n",
+    "\n",
+    "In our example, we might select six clusters:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "pcca = sim.msm.pcca(6)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "and depict the membership probabilities for each state"
```

Comment:
i thought this was super clear till now, but adding a third colorbar is a little confusing. i don't immediately have a solution thought. maybe just changing to a colorbar that is white for zero and a color, or black, for 1?,so it resembles occupancies more, would help

Reply:
agree

Reply:
blues look much better!